### ISSUE TITLE
New feature: onDeathTPTo

### ISSUE THREAD

**Issue by benda95280** [genboy/Festival#201]
Hello,

Is it possible to teleport player, if he died inside the area, to any chosen location on the world ?
(Outside the area)

Thanks.

**Comment by genboy** [comment]
@benda95280 yeah, i might add a onDeath event to the area's to put commands on. There are many more events to consider, i will make a priority schedule for next developments. Thank you for the creative input cheers
